src/eva/vision/data/wsi/backends/openslide.py

```diff
@@ -32,8 +32,28 @@ def level_downsamples(self) -> Sequence[float]:
     @override
     def mpp(self) -> float:
         # TODO: add overwrite_mpp class attribute to allow setting a default value
-        x_mpp = float(self._wsi.properties["openslide.mpp-x"])
-        y_mpp = float(self._wsi.properties["openslide.mpp-y"])
+        if self._wsi.properties.get("openslide.mpp-x") and self._wsi.properties.get(
+            "openslide.mpp-y"
+        ):
+            x_mpp = float(self._wsi.properties["openslide.mpp-x"])
```

Comment:
We can use `openslide.PROPERTY_NAME_MPP_X` and `openslide.PROPERTY_NAME_MPP_Y` instead of the strings.